FRAuth/FRAuth/DeviceBinding/ApplicationPinDeviceAuthenticator.swift

```diff
@@ -22,15 +23,34 @@ open class ApplicationPinDeviceAuthenticator: DeviceAuthenticator, CryptoAware {
     var cryptoKey: CryptoKey?
     /// AppPinAuthenticator to take care of key generation
     var appPinAuthenticator: AppPinAuthenticator?
+    /// ApplicationPinDeviceAuthenticatorDelegate for collection the Pin
+    public var delegate: ApplicationPinDeviceAuthenticatorDelegate?
     
+    init() {
```

Comment:
I would suggest to add a property in the initializer so that developers can pass their own collectors, since we give them the option to customize.

This could be optional and set a default value (the default collector) if no collector is passed

Reply:
Followed your suggestion